tests/test_command_bdist_dmg.py

```diff
@@ -0,0 +1,54 @@
+"""Tests for cx_Freeze.command.bdist_dmg."""
+
+from __future__ import annotations
+
+import sys
+from pathlib import Path
+from subprocess import run
+
+import pytest
+
+bdist_dmg = pytest.importorskip(
+    "cx_Freeze.command.bdist_dmg", reason="macOS tests"
+).bdist_dmg
+
+if sys.platform != "darwin":
+    pytest.skip(reason="macOS tests", allow_module_level=True)
+
+DIST_ATTRS = {
+    "name": "foo",
+    "version": "0.0",
+    "description": "cx_Freeze script to test bdist_mac",
+    "executables": ["hello.py"],
+    "script_name": "setup.py",
+    "author": "Marcelo Duarte",
+    "author_email": "marcelotduarte@users.noreply.github.com",
+    "url": "https://github.com/marcelotduarte/cx_Freeze/",
+}
+SAMPLES_DIR = Path(__file__).resolve().parent.parent / "samples"
+
+
+@pytest.mark.datafiles(SAMPLES_DIR / "simple")
```

Comment:
You can use your dmg sample here.

Reply:
Fixed in eded0cb4268111529354de18cdf6102af46ec1f3